packages/stats-validator-plugin-webpack/README.md

```diff
@@ -23,7 +23,7 @@ This package contains rules to validate a webpack bundle.
           // ensures that the build time has not exceeded the limit (10 sec)
           '@statoscope/webpack/build-time-limits': ['error', 10000],
           // ensures that bundle doesn't use specified modules
-          '@statoscope/webpack/restricted-modules': ['error', [/some-package\/dist/some-module\.js/]],
+          '@statoscope/webpack/restricted-modules': ['error', [/some-module\.js/,]],
```

Comment:
The main thing that I wanted to show by this example is that we can restrict using a specific module from any package and not restrict a whole package
For example `/\/lodash\/dist\/min\.lodash\.js/`

Reply:
@smelukov Thank you for the clarification. See it all right?